recipes/wslay/all/patches/0001-msvc-support.patch

```diff
@@ -0,0 +1,74 @@
+diff --git a/.gitignore b/.gitignore
```

Comment:
That's a great patch, you could open a PR to the author.

Reply:
It's already done, I just used the differential of this PR https://github.com/tatsuhiro-t/wslay/pull/36/files.

Reply:
Thank you!